lib/dotnet-parser.js

```diff
@@ -64,6 +64,11 @@ module.exports = {
       version: '0.0.0',
       packageFormatVersion: 'nuget:0.0.0',
       dependencies: {},
+      meta: {
+        targetFrameworks: dotnetVersions.map(function(framework) {
+          return framework.original;
```

Comment:
This looks weird for me. 1 tree = 1 target framework. But you have array here. 
I think `parse` should return multiple trees in the future, but for now you can return first TF from that array.

Reply:
👍